conf.yaml

```
@@ -127,8 +127,8 @@ contents:
             prefix:     en/welcome-to-elastic
             current:    *stackcurrent
             index:      welcome-to-elastic/index.asciidoc
-            branches:   [ 8.4 ]
-            live:       [ 8.4 ]
+            branches:   [ 8.5 ]
+            live:       [ 8.5 ]
```

For now, I think 8.5 wouldn't be considered "live"/"active", so I'd leave for now:
```suggestion
            live:       [ 8.4 ]
```